utils/localStorage.js

```diff
@@ -0,0 +1,71 @@
+function removeQueryFromLocalStorage(storageName, term) {
+    var queries = JSON.parse(window.localStorage.getItem(storageName));
```

Comment:
I'd suggest to create a function for both get and set items, where are you using `JSON.parse` consider to add try/catch because if for some reason it throws an error it won't break the component.
```js
function saveSuggestionQueries(storageName, queries) {
  window.localStorage.setItem(storageName, JSON.stringify(queries));
}

function getSuggestionQueries(storageName) {
  try {
    return JSON.parse(window.localStorage.getItem(storageName));
  } catch(ignored) {
    return [];
  }
}
```